bin/kano-profile-gui

```diff
@@ -164,6 +167,9 @@ class MainWindow(ApplicationWindow):
 if __name__ == '__main__':
     args = docopt.docopt(__doc__)
 
+    if is_internet():
+        run_cmd('sudo kano-content sync')
```

Comment:
Is there not a python function in kano-content which can do the sync, rather than having to launch it as a subprocess?